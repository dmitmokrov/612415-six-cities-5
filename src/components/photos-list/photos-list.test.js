import React from 'react';
import renderer from 'react-test-renderer';
import PhotosList from './photos-list';

const photos = [
  `img/1.jpg`,
  `img/2.jpg`,
  `img/3.jpg`,
  `img/4.jpg`
];

it(`Should PhotosList render correctly`, () => {
  const tree = renderer
    .create(<PhotosList
      photos={photos}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
