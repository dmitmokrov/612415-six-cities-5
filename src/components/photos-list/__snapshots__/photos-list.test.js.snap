// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PhotosList render correctly 1`] = `
Array [
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/1.jpg"
    />
  </div>,
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/2.jpg"
    />
  </div>,
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/3.jpg"
    />
  </div>,
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/4.jpg"
    />
  </div>,
]
`;
